interfaces/pivy_common_typemaps.i

```diff
@@ -76,7 +76,7 @@ SWIGEXPORT PyObject *
 cast(PyObject * self, PyObject * args)
 {
   char * type_name;
-  int type_len;
+  Py_ssize_t type_len;
```

Comment:
@looooo this change makes sense and it's especially this line that fixes the crash because the memory address of it is passed to PyArg_ParseTuple.
On 64-bit systems a Py_ssize_t usually is defined as a 64-bit type (on Posix systems as ssize_t (aka long) and non-Posix systems as intptr_t) but an int is only 32-bit and thus the passed memory block is too small.